cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,27 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
 
         // This test is being temporarily throttled to the service 'news'.
         if (service === 'news') {
+          it('should have a visible play button and duration', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media) {
+                const aresMediaBlocks = media.model.blocks[1].model.blocks[0];
+                const { durationISO8601 } = aresMediaBlocks.model.versions[0];
+
+                cy.get('div[class^="StyledVideoContainer"]').within(() => {
+                  cy.get('button')
```

Comment:
At the moment there is only one `<button>` inside of `StyledVideoContainer`, but that could change. To prevent this test from breaking in future, can I suggest we select the Play Button here? E.G.:
```
cy.get('div[class^="StyledPlayButton"]')
```

Reply:
I think what's there is fine for now, we can revise/refine it later on if need be.